README.md

```diff
@@ -60,6 +58,7 @@ Dev Dependencies
 - eslint-plugin-react (client only)
 - eslint-plugin-node (server only)
 - Prettier
+- Jest
 
```

Comment:
Why Jest Nalle...why not something else? like Chai..or Mocha..

Reply:
We chose to go with Jest because of two reasons: It's out of the box code coverage for react.js and it's fast testing speed. Even though it does not offer full flexibility, our use case for it does require highly complex tests.